bindings_test.go

```diff
@@ -719,30 +719,75 @@ func testBindingArray(t *testing.T, bulk bool) {
 
 func TestBulkArrayBinding(t *testing.T) {
 	runDBTest(t, func(dbt *DBTest) {
-		dbt.mustExec(fmt.Sprintf("create or replace table %v (c1 integer, c2 string)", dbname))
+		dbt.mustExec(fmt.Sprintf("create or replace table %v (c1 integer, c2 string, c3 timestamp_ltz, c4 time)", dbname))
+		now := time.Now()
+		someTime := time.Date(1, time.January, 1, 12, 34, 56, 123456789, time.UTC)
 		numRows := 100000
 		intArr := make([]int, numRows)
 		strArr := make([]string, numRows)
+		ltzArr := make([]time.Time, numRows)
+		timeArr := make([]time.Time, numRows)
 		for i := 0; i < numRows; i++ {
 			intArr[i] = i
 			strArr[i] = "test" + strconv.Itoa(i)
+			ltzArr[i] = now
+			timeArr[i] = someTime
 		}
-		dbt.mustExec(fmt.Sprintf("insert into %v values (?, ?)", dbname), Array(&intArr), Array(&strArr))
+		dbt.mustExec(fmt.Sprintf("insert into %v values (?, ?, ?, ?)", dbname), Array(&intArr), Array(&strArr), Array(&ltzArr, TimestampLTZType), Array(&timeArr, TimeType))
```

Comment:
what about other timestamp types? like TZ or NTZ? Don't we want to test these?

Reply:
Ok, added tz, ntz, date and binary.